rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -45,6 +45,11 @@ foreach(_extension IN LISTS AMENT_EXTENSIONS_rosidl_generate_interfaces)
   endif()
 endforeach()
 
+if("${_typesupport_impls} " STREQUAL " ")
+  message(WARNING "No valid typesupport implementations found.")
```

Comment:
The message should explicitly mention the Python generator since that is not obvious when reading the message in the CMake output.